components/kyma-environment-broker/internal/process/provisioning/connectivity_bind_test.go

```diff
@@ -1,99 +1,120 @@
 package provisioning
 
 import (
-	"encoding/json"
 	"testing"
 
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/connectivity_bind"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/provisioning/automock"
 	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/servicemanager"
-
-	"github.com/stretchr/testify/assert"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/stretchr/testify/require"
 )
 
-const connectivityServiceKey = `
-	{
-		"clientid" : "clientid314159265359",
-		"clientsecret": "Y2xpZW50c2VjcmV0Cg==",
-		"connectivity_service":
+func TestConnectivityBind_Run(t *testing.T) {
+	t.Run("Should succesfully apply overrides", func(t *testing.T) {
```

Comment:
There are two logic execution paths in tested step, dependent on the value of  `operation.Connectivity.Instance.Provisioned` We should test them both, that means first and second run. 

Also we could test the behaviour of retry after UpdateOperation is failed.